src/app/[locale]/specs/account/page.mdx

```diff
@@ -0,0 +1,90 @@
+
+The general expectation is that downstream services will not list an account as `active` if it is inactive at an upstream service. However, the concept of “upstream” is informal, and services may need to define their own policies for some corner-cases. For example, an AppView might consume from multiple independent relay providers, who report differing statuses for the same account, due to policy differences. If the network connection between a relay and PDS is disruptive, they may report different statuses for the same account. 
+
+If there is doubt about about an account’s general status in the network, the account’s current active PDS host can be queried using the `com.atproto.sync.getRepoStatus` endpoint. If the account is not `active` there, it is generally expected to not be active elsewhere, especially in the case of user-initiated actions (deactivation or deletion).
+
+### Content Deletion
+
+Separate from overall account status, individual pieces of content may be removed. When accounts remove records from their public repositories, hosting services should remove public access to that content.
+
+Applications which would benefit from explicit “tombstones” (indications that content previously existed) should explicitly design them in to Lexicon schemas. In the absence of explicit application-level tombstones, services are expected not to differentiate between content which has never existed and content which has been entirely deleted. Note that this is different from acknowledging account-level deletion, which is encouraged.
+
+## PDS Account Migration
+
+At a high level, the current PDS host for an account is indicated in the account’s DID document. If a DID document is updated to point at a new PDS host, the account status is `active` at that host, and there is a functioning repository (with valid signature and commit `rev`), then the account has effectively been migrated.
+
+At a protocol level, this is how account migration works. And in some situations, when any previous PDS instance is unavailable or has inactive account status, getting in to the above state may be the only account recovery path available.
+
+However, in the common case, when there is an active account on a functional current PDS, a more seamless account migration process is possible. It can summarized in a few high-level steps:
+
+- creating an account on the new PDS (which may start with `active` false, such as `deactivated`)
+- migrating data from the old PDS to the new PDS
+- updating identity (DID document) to point to the new PDS and using a new atproto signing key. For PLC DIDs, usually involves updating PLC rotation keys as well.
+- updating account status on both PDS instances, such that old is inactive, and new is active
```

Comment:
Deactivating the account on the old PDS is recommended but not strictly required, eg for the cases when it's down or malicious, right?

Reply:
correct; there also isn't really a mechanism to "enforce" it being required.

it is good to do because it makes it less likely for old PDS to emit `#commit` events by accident (eg, a cronjob or integration might have valid auth against old PDS and result in repo writes, which basically fork repo state and makes it critical for relays et al to have tracked account state correctly). it also resolves issues with multiple apps/devices and user confusion; and makes it more likely for the old PDS to be able to "clean up". We could have recommended "delete" old PDS account, but that felt risks (eg, it purges repo CAR and blobs, feels good to keep those copies around at least for a while for possible recovery).